10-fixtures.md

```diff
@@ -104,3 +104,61 @@ In addition, the setup and teardown functions will be automatically called for
 _every_ test in a given file so that each begins and ends with clean state.
 (Pytest has its own neat [fixture system](http://pytest.org/latest/fixture.html#fixture)
 that we won't cover here.)
+
+
+## Unittest
+
+The `unittest` module also provides this functionality for us with the `setUp` and `tearDown` methods that allow you to define instructions that will be executed before and after each test method (note the capitalisation of these names, which is important).
```

Comment:
perhaps you can use the notion of "fixtures" here, so that the student recognize the vocabulary that they just learned in the previous lesson?